maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -885,7 +885,7 @@ public void promptForDbCredentialsIfMissing(Server server) {
         //set password
         String dbPassword = promptForPasswordIfMissingWithDefault(
                 "Please specify database password (-D%s)",
-                server.getDbPassword(), "dbPassword", " ");
+                server.getDbPassword(), "dbPassword", "");
```

Comment:
Solution is acceptable. However since it doesn't allow white space only passwords (which users are unlikely do anyway), we must explicitly mention in OpenMRS SDK documentation.